app/models/promo_registration.rb

```diff
@@ -1,11 +1,38 @@
 class PromoRegistration < ApplicationRecord
   has_paper_trail
+  # self.per_page = 15
+
+  KINDS = ["channel", "owner", "unattached"].freeze
```

Comment:
nit:

```
CHANNEL = "channel".freeze
OWNER = "owner".freeze
UNATTACHED = "unattached".freeze
KINDS = [CHANNEL, OWNER, UNATTACHED].freeze
```

Reply:
Would you also add comments to explain what these mean (from a perspective of us looking back at this code 1 year from now)

Reply:
re nit:  What's the advantage of setting the constants individually?

I will add comments.

Reply:
It's just that when we use strings like "unattached" elsewhere in the code, it's not 100% certain that it's right (e.g. what if someone says "detached" or "dangling" etc.). Using misnamed constants will appropriately throw an exception in runtime due to the constant not being found.